paymentsheet/src/test/java/com/stripe/android/customersheet/state/DefaultCustomerSheetLoaderTest.kt

```diff
@@ -117,176 +107,17 @@ class DefaultCustomerSheetLoaderTest {
             )
         )
         assertThat(state.validationError).isNull()
-
-        val mode = elementsSessionRepository.lastParams?.initializationMode
-            as PaymentSheet.InitializationMode.DeferredIntent
-        assertThat(mode.intentConfiguration.paymentMethodTypes)
-            .isEmpty()
-    }
-
-    @Test
-    fun `load should return expected result when customer adapter has paymentMethodTypes`() = runTest {
```

Comment:
Do we have tests for these elsewhere?

Reply:
They are covered in the `CustomerAdapterDataSource` since it's specific to `CustomerAdapter`.